backend/dataall/cdkproxy/cdk_cli_wrapper.py

```diff
@@ -38,6 +38,7 @@ def aws_configure(profile_name='default'):
         encoding='utf-8',
         capture_output=True
     )
+    creds = None
```

Comment:
I think this is to pre-defined a variable, in case the cmd ran failed and return creds will cause 'unbound variable' error.

Reply:
@dlpzx I think we can check it in deploy_cdk_stack functions. At line 71, we could do like this:
```
if ENVNAME not in ['local', 'dkrcompose']:
    creds = aws_configure()
```

Reply:
> I think this is to pre-define a variable, in case the cmd ran failed and return creds will cause 'unbound variable' error.

Exactly, 

And yes you are right, let's add the "'dkrcompose'" environment

Reply:
@dlpzx Thanks!